package.json

```diff
@@ -148,15 +147,14 @@
     "backoff": "^2.5.0",
     "electron-is-dev": "^1.2.0",
     "electron-log": "^4.1.2",
-    "electron-spellchecker": "^2.2.1",
     "electron-updater": "^4.3.1",
     "electron-window-state": "^5.0.3",
     "escape-html": "^1.0.3",
     "fs-extra": "^9.0.0",
     "i18n": "^0.9.1",
+    "nan": "^2.14.0",
```

Comment:
Any need of this?

Reply:
Inherited from #928 I'll check the package once

Reply:
It’s required to work around CharlieHess/node-mac-notifier#19.